README.md

```diff
@@ -53,14 +53,68 @@ On OSX/Windows you can't run Docker natively. To allow the Makefile to write the
 Run `make test` like this: `LOCAL_USER_ID=1000 LOCAL_GROUP_ID=100 make test-containerized`
 
 
+
+## IPv6 Usage
```

Comment:
I added a note to the README.md, https://github.com/projectcalico/libnetwork-plugin/pull/142/files#diff-04c6e90faac2675aa89e2176d2eec7d8R59. 

Docker always has IPv4 enabled as far I can tell.